common/djangoapps/student/migrations/0001_squashed_0031_auto_20200317_1122.py

```diff
@@ -516,4 +516,10 @@ class Migration(migrations.Migration):
                 'abstract': False,
             },
         ),
+        migrations.AlterField(
+            model_name='courseenrollment',
+            name='course',
+            field=models.ForeignKey(db_constraint=False, on_delete=django.db.models.deletion.DO_NOTHING,
```

Comment:
Should on_deletes match original if they weren't there?  Was it CASCADE?  We should look at other on_deletes.

Reply:
I believe this matches what was in the original migration.